python/build-wheel-manylinux2014.sh

```diff
@@ -41,11 +41,6 @@ export JAVA_HOME="$java_home"
 #echo "build --config=manylinux2010" >> /root/.bazelrc
 echo "build --incompatible_linkopts_to_linklibs" >> /root/.bazelrc
 
-if [[ -n "${RAY_INSTALL_JAVA:-}" ]]; then
-  bazel build //java:ray_java_pkg
-  unset RAY_INSTALL_JAVA
-fi
```

Comment:
`python setup.py bdist_wheel` should trigger the `bazel build //java:ray_java_pkg`

Reply:
`bazel build //java:ray_java_pkg` has been executed here, so it's not necessary to execute it in `python setup.py bdist_wheel`.

Reply:
execute `bazel build //java:ray_java_pkg` in `python setup.py bdist_wheel` will execute three times for python 3.6, 3.7, 3.8

Reply:
But if we don't build java in python setup.py, the jars will not be copied in the python build process. This is a bug

Reply:
yeah, if we unset the `RAY_INSTALL_JAVA` the jars will not be copied to whl.

Reply:
I submitted a PR, could you help reivew it? #12456